### ISSUE TITLE
iOS : remove video compression on select from library

### ISSUE THREAD

**Issue by rudyZanotti** [apache/cordova-plugin-camera#732]
Hello,

I would like to know if it is possible to remove compression when selecting video on iOS. My users use a lot of videos and I would like to compress my videos while they can do other things.

thank you in advance 
Rudy

**Comment by PieterVanPoyer** [comment]
As far as I can see there is no compression done in the plugin when selecting a video on ios.

**Comment by josiaho** [comment]
I agree - I don't see any compression in the plugin either. However, the video is definitely significantly compressed on import. Can the plugin be modified to force transcoding bypass?
